src/test/scala/edu/ie3/simona/agent/grid/DBFSAlgorithmCenGridSpec.scala

```diff
@@ -8,18 +8,15 @@ package edu.ie3.simona.agent.grid
 
```

Comment:
Add test for multiple connection mechanisms that checks if 0 power is sent for the node that is not a slack node

Reply:
I enhanced the test to also include a double connection to the superior subgrid... This also means that the result values changed now.
Furthermore, I refactored the test further so that it became simpler, yet again.

I also adapted slack voltage communication to only send one message per subgrid, in the same way this PR introduces one sending a single message for grid power communication per subgrid.

Reply:
Ok, introducing the test for multiple slack nodes in powerflow computation revealed a shortcoming of the input data preparation, which is based on sets. Since sets have no determined order, at one point I had a test which worked on my PC and didn't in Jenkins. The solution here was to always pick the node with the smallest index (commit `b35c13ab`).